src/pages/Home/Homepage.css

```diff
@@ -0,0 +1,136 @@
+.homepage-container {
+    padding-left: 1.2rem;
+    padding-right: 1.2rem;
+    margin: auto;
+    max-width: 80rem;
+}
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+    margin-top: 4.375rem;
+}
+
+.header-column {
+    flex-basis: 50%;
+    min-width: 18rem;
+}
+
+.header-column img {
+    max-width: 100%;
+    padding: 50px 0;
+}
+
+.header-column h4 {
+    font-size: 40px;
+    line-height: 40px;
+    margin: 25px 0;
+}
+
+.featued-products-row {
+    display: flex;
+    justify-content: space-between;
```

Comment:
you cam use default css available from component library

Reply:
for `space-around` it was not readily available, so I used this. Thank you for bringing it to my notice.